templates/3b_Generic_Cluster_Cells.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    cell_som_cluster_count_avg_name = os.path.join("pixie", cell_output_dir, 'cell_som_cluster_count_avg.csv')
</pre><p>Number of SOM clusters isn't relevant for generic clustering, we should get rid of these. </p><p> </p><p>The useful visualization we'll want for this is the average value of each channel for the cells in each cluster</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I'm a little confused since <code>cell_som_cluster_count_avg_name</code> is the input to consensus clustering and the metaclustering step.</p><p> </p><p>Also, when you say average value of each channel for the cells in each cluster I'm assuming you mean the average values per cell of each expression column the user specifies? Does this need to be done per cell SOM/meta cluster as well or just per cell?</p>

Reply:
<p>Were we thinking about using the average value of each expression column per cell or cell SOM cluster as the inputs instead to ConsensusClustering?</p>

Reply:
Addressed by ensuring naming conventions are clear.